src/command/defaults/KillCommand.php

```diff
@@ -53,7 +54,7 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
 			return true;
 		}
 
-		$player->attack(new EntityDamageEvent($player, EntityDamageEvent::CAUSE_SUICIDE, 1000));
+		$player->attack(new EntityDamageEvent($player, EntityDamageEvent::CAUSE_SUICIDE, PHP_FLOAT_MAX));
```

Comment:
why not just INF?

Reply:
I'm dubious if PHP_FLOAT_MAX will actually work and not break anything, never mind INF.

Reply:
Yeah, INF is way too unpredictable. If you can verify that it works with PHP_FLOAT_MAX, I think that's tolerable, but my gut is telling me something is going to break with these insane damage amounts.